### ISSUE TITLE
Unable to add occurrences to an existing sample

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1772]
From Explore - My records, I used the Edit button to open an existing occurrence from a list sample. Normally that results in an editing page opening up that displays only the record being edited, plus an option to show the whole list and add to it.

In this case, the whole list is showing by default, and there is no option to add an extra occurrence - clicking on "Show the full list of records" has no effect.

It's also worrying that the stage attribute is showing as a blank for all the existing records.

![image](https://github.com/user-attachments/assets/d9a6617e-aa35-4b0b-9178-b5361abea101)

Maybe an issue arising from #1771 ?

**Comment by johnvanbreda** [comment]
Agreed - this was the same underlying issue as #1771. I've applied a temporary fix for now and will patch the main Indicia code properly tomorrow.

**Comment by johnvanbreda** [comment]
@BirenRathod there is a new release of iform which fixes this - it just needs an updated helper_base.php file. Please can you deploy that.

**Comment by BirenRathod** [comment]
@johnvanbreda I have updated the iForm module now.
